src/utils/FlowUtils.ts

```diff
@@ -64,27 +64,41 @@ export function filledSameDeposit(fillA: Fill, fillB: Fill): boolean {
 
 // Ensure that each deposit element is included with the same value in the fill. This includes all elements defined
 // by the depositor as well as the realizedLpFeePct and the destinationToken, which are pulled from other clients.
-export function validateFillForDeposit(fill: Fill, deposit?: Deposit, fillFieldsToIgnore: string[] = []): boolean {
```

Comment:
I think we can make some simplifications for v3 types here - I've proposed that in a separate PR: https://github.com/across-protocol/sdk-v2/pull/549/files.